.github/workflows/ci-mlir.yml

```
@@ -50,7 +50,7 @@ jobs:
       run: |
         # Change directory so that xdsl-opt can be found during installation.
         cd xdsl
-        make venv
+        UV_PROJECT_ENVIRONMENT=.venv uv sync --extra gui --extra dev --extra jax --extra riscv --extra wgpu --extra onnx
```

Instead of this, could we still go through the Makefile to ensure everything stays aligned

```suggestion
        VENV_EXTRAS="--extra gui --extra dev --extra jax --extra riscv --extra wgpu --extra onnx" make venv
```